mlir/lib/Dialect/Rock/Transforms/AccelEmitter.cpp

```diff
@@ -108,8 +132,8 @@ AccelEmitterParams MfmaEmitter::initAccelEmitterParams(
   params.mRepeats = mfmaGroup.getMRepeats(mPerWave);
   params.nRepeats = mfmaGroup.getNRepeats(nPerWave);
   params.nResultVectors = mfmaGroup.getImms().size();
-  params.mPerAccel = mfmaGroup.getLenPerMfmaGroup(mPerWave);
-  params.nPerAccel = mfmaGroup.getLenPerMfmaGroup(nPerWave);
+  params.mPerAccel = mfmaAttr.mfmaNonKDim * mfmaGroup.getImms().size();
+  params.nPerAccel = mfmaAttr.mfmaNonKDim * mfmaAttr.blocksMfma;
```

Comment:
Is this right?

Reply:
I think we miscomputed those, but we didn't use those values anywhere in the code (setting them both to (0,0) produces no failures). 

But I  over-complicated this calculation: what I needed was `mPerWave/mRepeats` and `nPerWave/nRepeats`. I.e., how many rows/colums a given wave computes in a single iteration.